invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less

```diff
@@ -27,10 +28,11 @@ nav {
   color: #ccc;
 }
 
-.navbar-options {
-  :hover {
+li.navbar-options:hover{ 
+  background-color: rgba(255, 255, 255, 0.2);
+  a:hover{
```

Comment:
Isn't the outer rule enough to extend the background to the whole `li` element?

Reply:
apparently not, the [hyperline]() default hover was applying - so i had to add rule of ```transparent```. and text color inside the hyperlink.